packages/models/src/email/emailNotificationMessagePayload.ts

```diff
@@ -0,0 +1,7 @@
+import { CorrelationId, Email, EmailAddress } from "../index.js";
```

Comment:
I think sometimes this kind of circular import (importing from `../index.js` which itself exports definitions from this file) can cause issues, though not always. I'd suggest importing from the specific files the definitions come from, not from `../index.js`, though I'm not sure if it's a problem in this case.